spec/reports/cost_report_spec.rb

```diff
@@ -17,7 +17,7 @@
   let(:holding2) { build(:holding, ocn: c.ocns.first, organization: "smu") }
 
   before(:each) do
-    Settings.target_cost = 10
+    @old_cost = Settings.target_cost
```

Comment:
I don't recall specifically what was breaking here, but in general if we're monkeying with settings in tests we should ensure they get set back to their original values so that we don't end up with behavior that depends on the order we run the tests.